### PR TITLE
chore: fix file path

### PR DESCRIPTION
**Pull request by imranbarbhuiya**
**Please describe the changes this PR makes and why it should be merged:**
`unbuild` uses `.cjs` extension which is failing docs workflows. But how it was passing earlier?
**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by iCrawl** [comment]
Looks like something else went wrong here anyway lol

```
Error: File not found: '/home/runner/work/discord.js/discord.js/./packages/actions/src/formatTag/../../dist/formatTag/index.js'
```

File: packages/actions/src/formatTag/action.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ outputs:
     description: 'The semver string that was extracted from this tag'
 runs:
   using: node16
-  main: ../../dist/formatTag/index.js
+  main: ../../dist/formatTag/index.cjs
```
Comment (iCrawl) [inline packages/actions/src/formatTag/action.yml:14]:
```suggestion
  main: ../../dist/formatTag/index.mjs
```

Can't we use `.mjs` instead here?

Comment (imranbarbhuiya) [reply packages/actions/src/formatTag/action.yml:14]: yeah, we can. Btw do we need cjs build for `/actions`? iirc when using tsup, we were using mjs only.

Comment (iCrawl) [reply packages/actions/src/formatTag/action.yml]: Yeah we don't in this case.

**Pull request merged**